ansible/roles/stig/defaults/main.yml

```diff
@@ -75,11 +75,13 @@ controls:
   - toss_04_020060
   - toss_04_020070
   - toss_04_020120
+  - toss_04_020140
```

Comment:
Did you / can you double check that we have all of the controls in both this list for `controls:` and also the list, and have all of them set to true [below](https://github.com/LLNL/toss-stig/blob/ddd1f4f6b575460762fcea7252ff6cd336e79bc6/ansible/roles/stig/defaults/main.yml#L232) ?

Reply:
Yup, I will do that. I'm working on that and the feedback from the review on Tuesday [over here](https://github.com/LLNL/toss-stig/pull/41)